test/cmdlineTests/standard_import_asm_json_immutable_references/output.json

```diff
@@ -8,7 +8,16 @@
                         "linkReferences": {},
                         "object": "<BYTECODE REMOVED>",
                         "opcodes":"<OPCODES REMOVED>",
-                        "sourceMap":"<SOURCEMAP REMOVED>"
+                        "sourceMap":"<SOURCEMAP REMOVED>",
+                        "subAssemblyOffsets": {
+                            "subs": [
+                                {
+                                    "isCreation": false,
```

Comment:
By the way, I wonder why this isn't `true`. Does exporting and reimporting asm JSON lose the creation flag? I so, this should be reported as a separate bug.